src/Modules/LearningTasks/Tutor.LearningTasks.API/Public/IAccessServices.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Tutor.LearningTasks.API.Public;
+
+public interface IAccessServices
+{
+    bool IsCourseOwner(int courseId, int instructorId);
```

Comment:
Why do we need this public API? Can't we just use the `IOwnershipValidator` wherever this interface is used?

If this is a mistake, don't forget to remove the interface and the service implementing it.